src/test/java/janet/EventTest.java

```diff
@@ -0,0 +1,37 @@
+package janet;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Based on [this guide](https://se-education.org/guides/tutorials/intellijCodeStyle.html#tweak-import-order), I think static imports should be ordered first.